crates/lua51_sys/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "lua51_sys"
+version = "0.1.0"
+keywords = ["bindings", "lua"]
+categories = ["external-ffi-bindings"]
+authors = ["Michael Pollind <mpollind@gmail.com>"]
+description = "Bindings to lua's C API"
+repository = "https://github.com/pollend/lua-sys"
```

Comment:
If this is living in the `rlua` repo then this repository link isn't right.

Reply:
opps, how would you want the cargo to look like? you can commit to this PR if you have the github cli.

Reply:
Well the repository should point here, and the names need updating as I've said in the other comment.
I've just checked and when publishing to crates.io there can't be any `path =` dependencies when publishing, so these do need to have all the right metadata and will be published as their own crates.

I admit to being a little unsure about what's right for the `authors` field.  I think if it were me doing it I'd leave it out (it's not actually used for much) but I don't feel strongly about it.